hurricane/webhooks/base.py

```diff
@@ -69,10 +70,12 @@ def run(self, url: str, status: WebhookStatus, error_trace: str = None, close_lo
                 self._callback_webhook_exception_check, url=url, close_loop=close_loop, loop=loop
             )
             fut.add_done_callback(callback_wrapper)
+        if not url and close_loop:
+            logger.warning("No webhook can be sent, as a url is not specified")
```

Comment:
I would phrase it as `"No webhook can be sent, as no url is specified"`, what do you think?